packages/design-system-react-native/src/components/BadgeCount/README.md

```diff
@@ -0,0 +1,141 @@
+# BadgeCount
+
+The `BadgeCount` component is a small, rounded badge used to display numerical values such as notifications, message counts, or other indicators. It supports different sizes and has a customizable maximum display value.
```

Comment:
Suggestion: Let's align the description with Figma to improve alignment across platforms

```suggestion
`BadgeCount` is a numeric indicator of unread messages or notifications on an app or UI element.
```

![Screenshot 2025-03-12 at 4 05 08 PM](https://github.com/user-attachments/assets/cceff7ad-7eca-4998-bd37-e0b152f1e785)